{"name": "insights.specs.Specs.postgresql_log", "exec_time": 1.6689300537109375e-05, "errors": ["Traceback (most recent call last):\n  File \"/var/lib/insights/newest.egg/insights/core/plugins.py\", line 101, in invoke\n    return self.component(broker)\n  File \"/var/lib/insights/newest.egg/insights/core/spec_factory.py\", line 798, in __call__\n    raise ContentException(\"[%s] didn't match.\" % ', '.join(self.patterns))\ninsights.core.exceptions.ContentException: [/var/opt/rh/rh-postgresql12/lib/pgsql/data/log/postgresql-*.log] didn't match.\n", "Traceback (most recent call last):\n  File \"/var/lib/insights/newest.egg/insights/core/plugins.py\", line 101, in invoke\n    return self.component(broker)\n  File \"/var/lib/insights/newest.egg/insights/core/spec_factory.py\", line 798, in __call__\n    raise ContentException(\"[%s] didn't match.\" % ', '.join(self.patterns))\ninsights.core.exceptions.ContentException: [/var/lib/pgsql/data/pg_log/postgresql-*.log] didn't match.\n"], "results": null, "ser_time": 4.5299530029296875e-06}